recipes/scmdata/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: scmdata
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/a9/71/9ea5101e0a2689d0fb8ddbd6c6db9ad77144292f9cedc5a7d78b9a0faddb/scmdata-{{ version }}.tar.gz
```

Comment:
If you run `conda-skeleton`, in `meta.yaml` you will have:
```
{% set name = "scmdata" %}
{% set version = "0.2.1" %}

package:
  name: "{{ name|lower }}"
  version: "{{ version }}"

source:
  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
  sha256: "c57adea051c04eb11df4b0b03895474eed130a6b5598d5639a2443d5b9d1b8b7"
```
so please modify your recipe accordingly.

Reply:
Please note that the url starts with `https://pypi.io/packages/source/` instead of `https://files.pythonhosted.org/packages`.

Reply:
great many thanks (fyi if I do `conda skeleton` I get the error message `FileNotFoundError: [Errno 2] No such file or directory: '/home/conda/staged-recipes/build_artifacts/work'`, I have no idea why...)